articles/welcome.md

```diff
@@ -1,62 +1,55 @@
 
-The current documentation includes the following topics.
+Implement [Grover's search algorithm](xref:microsoft.quantum.quickstarts.search) in just a few lines of code.
 
-* [Microsoft Quantum Development kit release notes](xref:microsoft.quantum.relnotes).
-* [Quantum computing concepts](xref:microsoft.quantum.concepts.intro) includes topics such as the relevance of linear algebra to quantum computing, the nature and use of a qubit, how to read a quantum circuit, and more.
-* [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.install) describes how to quickly set up your quantum development environment from any platform (Windows 10, Linux, and Mac) using Visual Studio, Visual Studio Code or the command line.  
-* [Quickstart - your first quantum program](xref:microsoft.quantum.write-program) walks you through how to write an application that creates a quantum entanglment state in the Visual Studio development environment. You'll learn how to define a Q# operation, call the Q# operation using C#, and how to execute your quantum algorithm.
-* [Managing quantum machines and drivers](xref:microsoft.quantum.machines) describes how quantum algorithms are executed, what quantum machines are available, and how to write a non-Q# driver for the quantum program.
-* [Quantum development techniques](xref:microsoft.quantum.techniques.intro) specifies the core concepts used to create quantum programs in Q#. Topics include file structures, operations and functions, working with qubits, and some advanced topics.
-* [Q# standard libraries](xref:microsoft.quantum.libraries.standard.intro) describes the operations and functions that support both the classical language control requirement and the Q# quantum algorithms. Topics include control flow, data structures, error correction, testing, and debugging. 
-* [Q# language reference](xref:microsoft.quantum.language.intro) details the Q# language including the type model, expressions, statements, and compiler use.
-* [For more information](xref:microsoft.quantum.more-information) contains specially selected references to deep coverage of quantum computing topics.
-* [Quantum trace simulator reference](xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators) contains reference material about trace simulator entities and exceptions.
-* [Q# library reference](xref:microsoft.quantum.standardlibsintro) contains reference information about library entities by namespace.
+Try one of the [Quantum Katas](xref:microsoft.quantum.overview.katas), short training exercises to teach you how to program in Q# and learn quantum computing.
+
+## Learn about the libraries
+
+Included in the QDK are a set of libraries that abstract scenario specific functionality, to make it easier for you to write quantum programs that target that scenario. For example, [Quantum Chemistry](https://cloudblogs.microsoft.com/quantum/2018/12/04/simulating-nature-with-the-new-microsoft-quantum-development-kit-chemistry-library/
```

Comment:
As written, this seems to create the impression that the Q# libraries are an advanced feature that only pertain to domain-specific applications. While the domain-specific libraries do exist and are critical, all Q# development will make use of the standard libraries as well, just as .NET developers make use of the BCLs and Python developers use the Python standard libraries.

I'd suggest that we should make sure that the learning paths that we offer with the refactored introduction don't position the Q# libraries as esoteric, but as part of how Q# offers a high-level quantum development experience to users.

Reply:
Sure, that makes sense.

Would this rewording be better:

In addition to the standard and numeric Q# libraries, the QDK includes libraries that abstract scenario specific functionality, to make it easier for you to write quantum programs that target that scenario. For example, [Quantum Chemistry](https://cloudblogs.microsoft.com/quantum/2018/12/04/simulating-nature-with-the-new-microsoft-quantum-development-kit-chemistry-library/

Reply:
I think that works a lot better, yeah. Thanks!